ampd/src/commands/worker_address.rs

```diff
@@ -0,0 +1,39 @@
+use error_stack::{Report, ResultExt};
+use std::path::PathBuf;
+
+use crate::state::{load, StateUpdater};
+use crate::tofnd::grpc::{MultisigClient, SharableEcdsaClient};
+use crate::tofnd::Config;
+use crate::Error;
+use crate::PREFIX;
+
+pub async fn run(config: Config, state_path: PathBuf) -> Result<(), Report<Error>> {
```

Comment:
use either `AsRef<Path>` or `&Path`, `PathBuf` is only necessary to correctly parse the starting args. It's similar to `String` and `&str`, so `PathBuf` makes a full copy of the content